crates/hotshot-signature-key/src/bn254/bn254_priv.rs

```diff
@@ -54,12 +54,9 @@ impl BLSPrivKey {
     }
 }
 
-#[allow(clippy::incorrect_partial_ord_impl_on_ord_type)]
```

Comment:
I'm good with this as long as it still works as intended ! I wonder why we implemented it this way to begin with

Reply:
I'm also wondering about that. In the beginning the implementation on this was very long ago, and later implementation just follow that. There are more discussions on whether we want to keep this [on zullip](https://espresso.zulipchat.com/#narrow/stream/311645-Engineering/topic/Clippy.20incorrect.20partial_cmp).